docs/guides/theming/Theming.stories.mdx

```diff
@@ -363,6 +365,67 @@ const MyComponent = () => {
   <CssVariables />
 </Story>
 
+## Default Props <a id="default-props" />
+
+Through our theme it is also possible to change the default props of a component. Let's say for example that the theme you want only uses squared
+shape avatars, you can use this feature to make your life easier by pre defining the default value of the prop `variant` to `square` in the `theme` `components` section like so:
+
+```tsx
+const newTheme = createTheme({
```

Comment:
Could we add the imports just for the sample to be similar to the other ones (and also users may find the imports useful)?

```
import { createTheme, HvProvider, HvAvatar } from "@hitachivantara/uikit-react-core";
```